security/backends/signals.py

```diff
@@ -0,0 +1,41 @@
+from django.dispatch import Signal, receiver
+
+from security.config import settings
+
+
+def get_backend_receiver(backend_name):
+    def backend_receiver(signal):
+        def _decorator(func):
+            def _wrapper(*args, **kwargs):
+                if settings.BACKENDS is None or backend_name in settings.BACKENDS:
```

Comment:
we can turn of backend for storing. So you can read from log but not write in it (good for tests)